### ISSUE TITLE
Trying to run at ubuntu server 20.04

### ISSUE THREAD

**Issue by thethinker990** [saveriomiroddi/zfs-installer#108]
Tried to run your command `GET https://git.io/JelI5 | sudo bash` on "ubuntu-20.04-live-server-amd64" (CTRL+ALT+F2) but after 10 minutes I get a failure 

```
bash: line1: unexptected EOF while looking for matching  `''
bash: line 4: syntax error: unexpected end of file
```

**Comment by saveriomiroddi** [comment]
Hello!

Can you try instead `curl -L https://git.io/JelI5 | sudo bash`?

I've recently updated the instructions, but forgot that Ubuntu Server doesn't provide GET by default (although that shouldn't case the error you've reported).

**Comment by thethinker990** [comment]
I get a timeout

`curl: (28) Failed to connect to raw.githubusercontent.com port 443: Connection timed out`

**Comment by thethinker990** [comment]
More or less offtopic:
I'm trying to do your script manually and get an error at
```
zpool create \
  "${encryption_options[@]}"\
  -o ashift=12 -O acltype=posixacl -O compression=lz4 -O dnodesize=auto -O relatime=on -O xattr=sa \
  -O normalization=formD \
  -O devices=off -O mountpoint=/ -R /mnt \
  rpool mirror ${first_disk_id}-part3 $rpool_mirror_arg
```
```
mountpoint '/mnt/' exists and is not empty
use '-m' option to provide a different default
```

**Comment by saveriomiroddi** [comment]
> I get a timeout
> 
> `curl: (28) Failed to connect to raw.githubusercontent.com port 443: Connection timed out`

Hello! This confirms a suspicion I had. You're experiencing a connection problem - this is not an issue with the script. The only other thing you can try is to use the expanded github address:

```sh
curl -L https://raw.githubusercontent.com/saveriomiroddi/zfs-installer/stable/install-zfs.sh | sudo bash
```

but other than this, I can't provide help, because the problem is not in the domain of the installer.

**Comment by saveriomiroddi** [comment]
> More or less offtopic:
> I'm trying to do your script manually and get an error at
> 
> ```
> zpool create \
>   "${encryption_options[@]}"\
>   -o ashift=12 -O acltype=posixacl -O compression=lz4 -O dnodesize=auto -O relatime=on -O xattr=sa \
>   -O normalization=formD \
>   -O devices=off -O mountpoint=/ -R /mnt \
>   rpool mirror ${first_disk_id}-part3 $rpool_mirror_arg
> ```
> 
> ```
> mountpoint '/mnt/' exists and is not empty
> use '-m' option to provide a different default
> ```
> 
> btw I am at this point: https://saveriomiroddi.github.io/Installing-Ubuntu-on-a-ZFS-root-with-encryption-and-mirroring/#prepare-the-disks
> 
> `-O overlay=on` didn't did the trick
> 
> `ls -l /mnt` just contains "boot", which is empty

I guess you're creating the `bpool` before the `zpool`. If that's _not_ the case, `rm -rf /mnt/boot`.

I need to close this issue. Sorry for that, but I can't provide any other support for this, since, again, it's not a problem with the script, and my resources are very limited.

For this type of issues, the ZFS mailing list can offer you help!

But my advice is to solve the internet connection issue and run the script. You can use several workarounds, for example, copying the script into pastebin and download it from there, or copy it onto a USB key from another computer, etc.etc.

**Comment by thethinker990** [comment]
> > I get a timeout
> > `curl: (28) Failed to connect to raw.githubusercontent.com port 443: Connection timed out`
> 
> Hello! This confirms a suspicion I had. You're experiencing a connection problem - this is not an issue with the script. The only other thing you can try is to use the expanded github address:
> 
> ```shell
> curl -L https://raw.githubusercontent.com/saveriomiroddi/zfs-installer/stable/install-zfs.sh | sudo bash
> ```
> 
> but other than this, I can't provide help, because the problem is not in the domain of the installer.

This did it. Why I don't came up with this idea.. 'doh
Thank you very much
